app/v2/controllers/V2MovementsController.scala

```diff
@@ -467,6 +468,13 @@ class V2MovementsControllerImpl @Inject() (
         EitherT.rightT(())
     }
 
+  def determineMaxPerPageCount(parameter: Option[Long]): Option[ItemCount] =
+    parameter match {
+      case Some(x) if x <= config.maxItemsPerPage => Some(ItemCount(x))
+      case Some(x) if x > config.maxItemsPerPage  => Some(ItemCount(config.maxItemsPerPage))
+      case None                                   => None
```

Comment:
This should also be limited to the max items per page.

Reply:
In that case do we need to remove ```defaultItemsPerPage```?  As there will not be any use of ```defaultItemsPerPage``` if I'll change this to ```maxItemsPerPage```

Reply:
Ah, right, yes, there's a default that gets set.

In that case, I'd like to see this logic modified because it's confusing to have this in two places like this. Choosing the page count should be done in one place, so we should move the default page count logic to here and have this just return an `Int`.

Reply:
Well `ItemCount`.